contrib/vagrant/deployments/coredns-controller.yaml

```diff
@@ -52,7 +52,7 @@ spec:
         kubernetes.io/os: linux
       containers:
       - name: coredns
-        image: k8s.gcr.io/coredns:1.7.0
+        image: k8s.gcr.io/coredns/coredns:v1.8.0
```

Comment:
Probably need to update https://github.com/cilium/packer-ci-build/blob/master/provision/pull-images.sh accordingly.

Reply:
Done https://github.com/cilium/packer-ci-build/pull/260